docs/openfaas-pro/sns-events.md

```diff
@@ -0,0 +1,89 @@
+# Trigger functions from AWS SNS messages
+
+With the SNS connector AWS SNS subscriptions can be used to trigger OpenFaaS functions.
```

Comment:
...from many different kinds of AWS services such as S3, DynamoDB, Redshift and so on.